package.json

```diff
@@ -22,11 +22,19 @@
   "license": "Apache-2.0",
   "peerDependencies": {
     "@emotion/css": "11.1.3",
-    "react-select": "5.2.1"
+    "react-select": "5.2.1",
```

Comment:
I think this needs to be a semver range like `^5.2.0` which will support up to `6.0.0` and prevent cli noise when running `yarn` in projects that consume this package.